CodeStarterCamp_Week2/main.swift

```diff
@@ -8,5 +8,19 @@
 
 import Foundation
 
-print("Hello, World!")
+let myLottoNumbers: Set<Int> =  [1, 2, 3, 4, 5, 6]
+let numbers = Array(1...45)
+var lotto: Set<Int> = []
 
+    while lotto.count < 6 {
+        lotto.insert(numbers.randomElement()!)
+    }
+
+func matchingLotto(myLottoNumbers: Set<Int>, lotto: Set<Int>) {
+    let result: Set<Int> = myLottoNumbers.intersection(lotto)
+    if result.isEmpty {
+        print("아쉽지만 겹치는 번호가 없습니다.")
+    }         else {
+        print("축하합니다! 겹치는 번호는 \(result)입니다!")
+    }
+}
```

Comment:
LGTM 👍👍👍

다만 겹치는 결과를 출력할 때 괄호를 출력하지 않고 숫자만 출력되도록 수정해 볼 수 있을까요?

또한 `else`사용을 하지 않고 return을 활용하여 코드를 작성해볼 수 있을것 같아요.

Reply:
```suggestion
func matchingLotto(myLottoNumbers: Set<Int>, lotto: Set<Int>) {
    let result: Set<Int> = myLottoNumbers.intersection(lotto)
    if result.isEmpty {
        print("아쉽지만 겹치는 번호가 없습니다.")
    }         
    return print(result)
}

matchingLotto(myLottoNumbers: myLottoNumbers, lotto: lotto)
```
요청주신대로 else사용을 하지 않고  return을 활용하여 숫자만 출력되도록 코드를 작성해봤습니다.